packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -538,7 +538,7 @@ const VirtualListBaseFactory = (type) => {
 					} else if (
 						directions.right && repeat ||
 						directions.left && Spotlight.move(direction) ||
-						(directions.up || directions.down) && (repeat || moveFocusStraight({id: this.props.spotlightId, direction}) && currentTarget.contains(Spotlight.getCurrent()))
+						(directions.up || directions.down) && focusableScrollbar && (repeat || moveFocusStraight({id: this.props.spotlightId, direction}) && currentTarget.contains(Spotlight.getCurrent()))
```

Comment:
I don't think that we can add this guard for `focusableScrollbar` quite like this, since it would mean we would never be able to prevent `onKeyDown` bubbling when the scroll button had focus via pointer and 5way up/down (on a vertical list) was pressed.
We want to consume `onKeyDown` events when focus remains within the list and bubble the event when focus would attempt to leave. This is the reason for the last conditional in this line: `currentTarget.contains(Spotlight.getCurrent())`, since focus remains in the container, we can prevent bubbling.